addon/components/polaris-avatar.js

```diff
@@ -25,11 +25,10 @@ const avatarImages = [
 
 const styleClasses = ['one', 'two', 'three', 'four', 'five', 'six'];
 
+@deprecateClassArgument
```

Comment:
This is neat, but what do you think of having a generic `deprecateArgument` decorator instead so that we have something reusable going forward? Maybe gets a bit too clunky having to pass the various deprecation properties (message, version etc.)?

Reply:
yeah, I def like how simple the usage with this is
but something we should keep in mind and maybe do in the future 👍